atomic_reactor/inner.py

```diff
@@ -492,6 +493,7 @@ def __init__(
         self.pipeline_run_name = pipeline_run_name
         self.source = source or DummySource(None, None)
         self.user_params = user_params or self._default_user_params.copy()
+        self.platforms_result = platforms_result
```

Comment:
Should we add a short dockstring also for `platforms_result` param?

Reply:
added